modules/ROOT/pages/sap/sap-s4hana-cloud-connector.adoc

```diff
@@ -1,35 +1,29 @@
-= SAP S4 HANA Cloud Connector - Mule 4
+= SAP S/4HANA OData Cloud Connector - Mule 4
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 :imagesdir: ../../assets/images/
 
-Support Category: Premium
+Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Premium]
 
-SAP S/4HANA Cloud Connector v1.1
+SAP S/4HANA Cloud Connector v1.2
 
-The Anypoint Connector for SAP S/4HANA Cloud provides full support for 
-the query, get, create, update, delete entities, and execute functions using the ODATA API exposed by SAP S/4HANA.
+Anypoint Connector for SAP S/4HANA OData Cloud (S/4HANA OData Cloud Connector) provides full support for 
+the query, get, create, update, delete entities, and execute functions using the OData API exposed by SAP S/4HANA.
 
-This connector lets you perform operations to:
+The connector lets you perform operations to:
 
-* Create, update, and delete entities.
-* Retrieve a single entity or query multiple entities.
-* Invoke a remote function.
+* Create, update, and delete entities
+* Retrieve a single entity or query multiple entities
```

Comment:
Do you suppose that this should be two list items, since both retrieve and query, and single and multiple, are different?